Update security_bulletin_faq.rst (#626)
AviatrixSystems/Docs: HowTos/security_bulletin_faq.rst

By default, data transfer is over a TCP connection with TLSv1.2 for encryption. Customers have the option to downgrade the TLS Version used due to internal dependency conflicts. You can configure this in Aviatrix Controller by clicking on Settings > Advanced > Security.

How does Aviatrix encrypt data in transit? 
--------------------------------------------------------------------------------------------
Aviatrix 6.5 and above implements the spiffee framework which use a PKI/X.509 strong encryption. You can read more about this framework at spiffee.io. 

How does Aviatrix handle security patch?
--------------------------------------------------------------------------------------------

> Security patch resolves software vulnerabilities and will be patched to the compatible software versions as stated in the release notes. When a patch is released, there will be a field notice to Aviatrix Controller via email. In addition, customer can send an email to securityalerts@aviatrix.com with a subject Subscribe, to subscribe to our customer mailing list. 

Reviewer: ```suggestion
A security patch resolves software vulnerabilities and will be applied to the compatible software versions as stated in the release notes. When a patch is released, there will be a field notice to Aviatrix Controller via email.
```

Reviewer 2: We should also remove the reference to SPIFFE

> How do I stay up to sate with the latest security vulnerabilities?

Reviewer: ```suggestion
How do I stay up to date with the latest security vulnerabilities?
```

--------------------------------------------------------------------------------------------

> We recommend customer to deploy the latest image, upgrading to the latest Software version, and staying on top of any security patch released. Security vulnerabilities is complex Aviatrix's policy is to address them continously in the latest releases. 

Reviewer: ```suggestion
We recommend customers to deploy the latest image, upgrading to the latest software version, and staying on top of any security patch released. Guaranteeing security against vulnerabilities is a sustained effort and it is Aviatrix's policy to address them continuously. 
```
